src/Resources/views/orderRefunds.html.twig

```diff
@@ -20,6 +20,7 @@
                     </div>
                 </div>
                 <form action="{{ path('sylius_refund_refund_units', {'orderNumber': app.request.attributes.get('orderNumber')}) }}" method="post">
+                    <input type="hidden" name="_csrf_token" value="{{ csrf_token(app.request.attributes.get('orderNumber')) }}" />
```

Comment:
Do we check this csrf token in action?

Reply:
Nope, thanks for noticing this. I don't know why we had this token in Plus, but I have added a validation now.